guides/common/modules/proc_installing-the-satellite-server-packages.adoc

```diff
@@ -3,38 +3,46 @@ ifdef::context[:parent-context: {context}]
 [id="Installing_Server_Packages_{context}"]
 = Installing {ProjectServer} Packages
 
+ifdef::foreman-el,katello,satellite[]
+* xref:#rhel-8[{RHEL} 8]
+* xref:#rhel-7[{RHEL} 7]
+endif::[]
 ifdef::foreman-el,katello[]
 * xref:#centos-8[CentOS 8]
 * xref:#centos-7[CentOS 7]
 endif::[]
-ifdef::foreman-el,katello[]
-* xref:#rhel-7[{RHEL} 7]
-endif::[]
 
 .Procedure
```

Comment:
Unrelated to this PR, but popped up in my head.

This "procedure" looks odd, as it is only rendered for the very first OS entry:
![image](https://user-images.githubusercontent.com/94284/172638313-1e627a50-90ad-4538-9c59-3351c004f992.png)